### DISCUSSION TITLE
Extreme framedrops in beatmap menu

### DISCUSSION THREAD

**Discussion by Zockerboii** [ppy/osu#17807] [Q&A] [answered]
### Type

Performance

### Bug description

When I am in the main menu osu lazer i have 70-90 fps. But if I go to the beatmap selector I have now just 20-30 fps. And ingame its like 60-70fps.

### Screenshots or videos

Main menu:
![osu_2022-04-13_20-16-09](https://user-images.githubusercontent.com/103599451/163244786-6e6f105b-3d58-41cc-88aa-3182218e6efe.jpg)

Selector:
![osu_2022-04-13_20-16-19](https://user-images.githubusercontent.com/103599451/163244882-b252708b-d77d-42c0-9bc3-0d6a98f2c2ec.jpg)

Ingame:
![osu_2022-04-13_20-17-24](https://user-images.githubusercontent.com/103599451/163244920-b258e55b-fe40-4a97-94c9-5eaf0221eb7d.jpg)

### Version

osu! 2022.408.0-lazer

### Logs

[database.log](https://github.com/ppy/osu/files/8483776/database.log)
[network.log](https://github.com/ppy/osu/files/8483777/network.log)
[performance.log](https://github.com/ppy/osu/files/8483778/performance.log)
[runtime.log](https://github.com/ppy/osu/files/8483779/runtime.log)
[updater.log](https://github.com/ppy/osu/files/8483780/updater.log)

**Comment by smoogipoo** [comment, marked as answer]
Performance is a constant work in progress. Make sure you have the most up-to-date drivers for your system.

For a GPU that is over a decade old, likely the only thing that's going to improve this is the recent work into Veldrid which would allow DirectX to be used.
